test/testtxref/test_Txref.cpp

```diff
@@ -708,14 +714,33 @@ TEST(TxrefTest, txref_decode_bip_examples) {
     RC_ASSERT(decodedResult.transactionPosition == pos);
 
 
-    // The following list gives valid (though strangely formatted) Bitcoin TxRef's
-    height = 0x71F69; pos = 0x89D; txref = "tx1:rjk0-uqay-z9l7-m9m";
+    // The following list gives valid (sometimes strangely formatted) TxRef's
+    height = 456789; pos = 1234; txref = "tx1:r29u-mqjx-putt-3p0";
     decodedResult = txref::decode(txref);
     RC_ASSERT(decodedResult.blockHeight == height);
     RC_ASSERT(decodedResult.transactionPosition == pos);
 
+    txref = "TX1R29UMQJXPUTT3P0";
+    decodedResult = txref::decode(txref);
+    RC_ASSERT(decodedResult.blockHeight == height);
+    RC_ASSERT(decodedResult.transactionPosition == pos);
 
-    // The following list gives properly encoded Bitcoin mainnet TxRef's with Outpoints
+    txref = "TX1R29umqJX--PUTT----3P0";
```

Comment:
this doesn't look valid.

Reply:
It was this "random" test that caused me to realize we could be lower-casing all input strings (as mentioned above).